library/src/main/java/com/mapzen/android/DataLayerType.java

```diff
@@ -0,0 +1,17 @@
+package com.mapzen.android;
+
+/**
+ * Created by sarahlensing on 7/27/16.
+ */
+public enum DataLayerType {
```

Comment:
This list of layers looks a lot like the layer name constants declared in `OverlayManager`.

https://github.com/mapzen/android/blob/2118e5059299318e4ec4d4aa5abf1b8c967b9a53/library/src/main/java/com/mapzen/android/OverlayManager.java#L33-L42

Rather than maintain two separate lists what if we deleted the constants in `OverlayManager` and modified `DataLayerType` to make the string value be the name of the layer?

```java
/**
 * Default data layers for bundled Mapzen stylesheets including Bubble Wrap, Cinnabar, and Refill.
 * Used to add client data layers to the map and optionally persist them on configuration change.
 */
public enum DataLayerType {
  CURRENT_LOCATION("mz_current_location"),
  POLYLINE("mz_default_line"),
  POLYGON("mz_default_polygon"),
  MARKER("mz_default_point"),
  ROUTE_START_PIN("mz_route_start"),
  ROUTE_END_PIN("mz_route_destination"),
  DROPPED_PIN("mz_dropped_pin"),
  SEARCH_RESULT_PIN("mz_search_result"),
  ROUTE_PIN("mz_route_location"),
  ROUTE_LINE("mz_route_line"),
  TRANSIT_ROUTE_LINE("mz_route_line_transit"),
  TRANSIT_ROUTE_LINE_STATION_ICON("mz_route_transit_stop");

  private final String name;

  DataLayerType(String s) {
    name = s;
  }

  public String toString() {
    return name;
  }
}
```